testing/include/ignition/common/testing/Utils.hh

```diff
@@ -0,0 +1,41 @@
+/*
+* Copyright (C) 2022 Open Source Robotics Foundation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+*/
+#ifndef IGNITION_COMMON_TESTING_UTILS_HH_
+#define IGNITION_COMMON_TESTING_UTILS_HH_
+
+#include <climits>
+#include <cstdint>
+#include <string>
+
+namespace ignition::common::testing
+{
+
+/////////////////////////////////////////////////
+/// \brief Get a random number based on an integer converted to string.
+/// \return A random integer converted to string.
+std::string getRandomNumber(int32_t _min = 0, int32_t _max = INT_MAX);
```

Comment:
`ign-math` has helper functions to return random numbers. I looks like the `getRandomNumber` is only used in the tests.

You could replace those instances with:
```
auto val1 = std::to_string(math::Rand::IntUniform(min,max));
```

Reply:
Good call, this got lifted straight out of a `test_config.h` somewhere, so probably not needed.

Reply:
[44cce83](https://github.com/ignitionrobotics/ign-common/pull/314/commits/44cce83d96a8a943132d4127258480d8b9e794fd)